modules/blender/README.md

```diff
@@ -0,0 +1,8 @@
+# Blender
+This module supports [Blender](https://blender.org).
+
+
+> [!IMPORTANT]
+> The Blender target will have no effect unless the Blender theme is properly
+> [enabled](https://docs.blender.org/manual/en/latest/editors/preferences/themes.html)
+> within Blender itself.
```

Comment:
Can we generate a configuration file defaulting the theme to Stylix? If this is not possible due to read-only issues, the regular configuration could be modified with a `home.activation` script. In that case, we might want to add an option to disable this option, in case people want the Stylix theme available but do not want Stylix to override the default theme everytime they rebuild their configuration.

Reply:
I could look into it but I will be pretty busy with my studies for the next few weeks so it might take a bit of time

Reply:
> I could look into it but I will be pretty busy with my studies for the next few weeks so it might take a bit of time

No worries - and if you have exams then good luck :))

Reply:
In that case, let's leave it for now

Reply:
okay! what are we missing then?